doc/python/interconnectivity.rst

```diff
@@ -10,6 +10,8 @@ Interconnectivity
     Describes a connection between two cells, defined by source and destination end points (that is pre-synaptic and post-synaptic respectively),
     a connection weight and a delay time.
 
+    A :class:`connection` is associated with the destination cell of the connection and its gid.
+
```

Comment:
Also this sentence, analogously to the one in the gap junction documentation, is not clear…